Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2039,6 +2026,7 @@ extension BrowserViewController: TabManagerDelegate {
     fileprivate func updateTabCountUsingTabManager(_ tabManager: TabManager, animated: Bool = true) {
         if let selectedTab = tabManager.selectedTab {
             let count = selectedTab.isPrivate ? tabManager.privateTabs.count : tabManager.normalTabs.count
+            toolbar?.updateTabCount(max(count, 1), animated: animated)
```

Comment:
I don't think this `max` should go here (and the two others below) because updateTabCount is open to anyone to call it, yet only this function knows how to safely call it. I assume the count label display code is common? If so that would be a better place to assert a min display value of 1.